payments-core/src/main/java/com/stripe/android/model/CardParams.kt

```diff
@@ -68,6 +68,16 @@ data class CardParams internal constructor(
      */
     var currency: String? = null,
 
+    /**
+     * Contains information about card networks used to process the payment.
```

Comment:
can you update the javadoc to match the format of the other fields? It looks like we typically start with the link to the generic docs and then add info about it being optional + the documentation